components/http_kernel/introduction.rst

```diff
@@ -85,11 +85,22 @@ is really simple and involves creating an
 :doc:`event dispatcher </components/event_dispatcher/introduction>` and a
 :ref:`controller resolver <component-http-kernel-resolve-controller>` (explained
 below). To complete your working kernel, you'll add more event listeners
-to the events discussed below::
+to the events discussed below
```

Comment:
Moved the caution to below, this also fixes the issue where the : refers to a caution instead of the intended code.